src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/Et1VettingController.java

```diff
@@ -140,6 +143,35 @@ public ResponseEntity<CCDCallbackResponse> et1HearingVenue(
         return getCallbackRespEntityNoErrors(caseData);
     }
 
+    /**
+     * About to Submit callback which handle the submission of data from the event into CCD and also generates a PDF
+     * of the ET1 Vetting data.
+     * @param userToken Used for authorisation
+     * @param ccdRequest CaseData which is a generic data type for most of the methods which holds ET1 case data
+     * @return caseData in ccdRequest
+     */
+    @PostMapping(value = "/et1VettingAboutToSubmit", consumes = APPLICATION_JSON_VALUE)
+    @Operation(summary = "Generates the PDF for ET1 Vetting.")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "200", description = "Accessed successfully"),
+        @ApiResponse(responseCode = "400", description = "Bad Request"),
+        @ApiResponse(responseCode = "500", description = "Internal Server Error")
+    })
+    public ResponseEntity<CCDCallbackResponse> et1VettingAboutToSubmit(
+            @RequestHeader(HttpHeaders.AUTHORIZATION) String userToken,
+            @RequestBody CCDRequest ccdRequest) {
+        log.info("ET1 CASE VETTING ABOUT TO START ---> " + ccdRequest.getCaseDetails().getCaseId());
```

Comment:
Prefer log.info("ET1 CASE VETTING ABOUT TO START ---> {}", ccdRequest.getCaseDetails().getCaseId());